pilot/pkg/networking/core/v1alpha3/httproute_test.go

```diff
@@ -398,10 +398,7 @@ func TestSidecarOutboundHTTPRouteConfig(t *testing.T) {
 			routeName:             "9000",
 			sidecarConfig:         sidecarConfig,
 			virtualServiceConfigs: nil,
-			expectedHosts: map[string]map[string]bool{
-				"bookinfo.com:9999": {"bookinfo.com:9999": true, "*.bookinfo.com:9999": true},
-				"bookinfo.com:70":   {"bookinfo.com:70": true, "*.bookinfo.com:70": true},
-			},
+			expectedHosts:         map[string]map[string]bool{},
```

Comment:
This sort of makes this test useless, should we change the sidecar config in the test to select a port at least?

Reply:
Will change

Reply:
I did not realize, the test case "sidecar config port that is in one of the service" below does the same - this is case IMO is also valid to test.